### DISCUSSION TITLE
Can I add custom release name for ply console while making internal release from code magic workflow ?

### DISCUSSION THREAD

**Discussion by aitzaz-dev** [codemagic-ci-cd/codemagic-docs#2657] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I made a release successfully using code magic workflow. But the release name was 1.0.0. I want to add custom release name (like: "Bug Fixing", "Some feature name") to track the releases easily in play console.

### Expected results

like: "Bug Fixing", "Some feature name"

### Actual results

1.0.0

**Comment by himesh-cm** [comment]
Hey @aitzaz-dev 

You can set the extra information in your release notes, which your users can see on the Play Store as shown in the example screenshot below.

![Screen Shot 2024-08-06 at 17 26 48 PM](https://github.com/user-attachments/assets/731d7219-fc4e-4756-85f2-be15bafdfa5f)

```
[
    {
        "language": "en-US",
        "text": "Bug Fixes"
    }
]
```

You can read more about it here in our documentation - https://docs.codemagic.io/yaml-notification/publish-release-notes/

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
